src/index.ts

```diff
@@ -141,10 +145,23 @@ export function withSpid(
     serviceProviderConfig,
     samlConfig
   );
-
-  return loadSpidStrategyOptions()
+  // If `startupIdpsMetadata` is provided, Spid Strategy options are loaded
+  // from that variable instead downloaded from remote idps metadata URL
+  return fromNullable(null)(appConfig.startupIdpsMetadata)
```

Comment:
why not using fromNullable from Option ?

Reply:
Fixed in 031a6f1